src/utils/queryStringify.ts

```diff
@@ -0,0 +1,7 @@
+const queryStringify = (data: Record<string, string>): string => {
+  const str = Object.keys(data).reduce((acc, key) => [...acc, `${key}=${data[key]}`], []);
+
+  return `?${str.join('&')}`;
+};
```

Comment:
Я думаю такой код будет понятнее
```suggestion
const queryStringify = (data: Record<string, string>): string => Object
            .entries(data)
            .map(([k, v]) => k + "=" + v)
            .join("&")
```